services/node_manager/src/main/java/com/futurewei/alcor/nodemanager/dao/NmmInfoRepository.java

```diff
@@ -0,0 +1,161 @@
+/*Copyright 2019 The Alcor Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+        you may not use this file except in compliance with the License.
+        You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+        Unless required by applicable law or agreed to in writing, software
+        distributed under the License is distributed on an "AS IS" BASIS,
+        WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+        See the License for the specific language governing permissions and
+        limitations under the License.
+*/
+package com.futurewei.alcor.nodemanager.dao;
+
+import com.futurewei.alcor.common.db.CacheException;
+import com.futurewei.alcor.common.db.CacheFactory;
+import com.futurewei.alcor.common.db.ICache;
+import com.futurewei.alcor.common.db.Transaction;
+import com.futurewei.alcor.common.db.repo.ICacheRepository;
+import com.futurewei.alcor.common.stats.DurationStatistics;
+import com.futurewei.alcor.web.entity.node.NmmInfo;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Repository;
+
+import javax.annotation.PostConstruct;
+import java.util.List;
+import java.util.Map;
+
+@Repository
+public class NmmInfoRepository implements ICacheRepository<NmmInfo> {
+    private static final Logger logger = LoggerFactory.getLogger(NmmInfoRepository.class);
+    private ICache<String, NmmInfo> cache;
+
```

Comment:
@pkommoju Please give a comment here to describe the content of the mapping, what is data for the String in cache?

Reply:
The string is the NCM Identifier assigned by the Admin.

Reply:
Added thye required explanation.